checkov/common/bridgecrew/image_scanning/twistlock_integration.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import os
+import platform
+import stat
+import requests
+
+from checkov.common.bridgecrew.platform_integration import bc_integration
+from checkov.common.bridgecrew.integration_features.base_integration_feature import BC_API_URL
+from checkov.common.util.dict_utils import merge_dicts
+from checkov.common.util.http_utils import get_auth_header, get_default_get_headers
+
+
+class TwistLockIntegration:
+    twistlock_base_url = f"{BC_API_URL}/vulnerabilities/twistlock"
```

Comment:
please change url to "vulnerabilities/docker_images"

Reply:
Done